### DISCUSSION TITLE
R68S重启后会出现BUG

### DISCUSSION THREAD

**Discussion by SLiang-liao** [coolsnowwolf/lede#9535] [R68S / 电犀牛专区]
R68S重启前如果插着Wan口的网线，路由器在重启后会出现断流现象。但是重启前拔掉网线，重启后插上，就有网络。

**Comment by Bigbogai** [comment]
同问：光猫桥接，国际版固件R68S拨号成功有网，网页重启之后拨号成功但无法上网，物理断电重启才能正常上网。

**Comment by nobodysclown** [comment]
我这也是一直断网，好像是DHCP 有bug。正常一段时间后设备获取的ip为169.254开头，无法联网
